style.css

```diff
@@ -10,19 +13,20 @@ html {
 	gap: 30px;
 	background-color: #BD4B4B;
```

Comment:
This colour is too close to your font colour and it fails a contrast check, try to get at least AA on here: https://webaim.org/resources/contrastchecker/